docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -27,6 +27,46 @@ kubectl label namespace default knative-eventing-injection=enabled
    with the appropriate permissions, you need to modify the
    `serviceaccount.yaml`.
 
+   ```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: events-sa
+  namespace: default
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: event-watcher
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - get
+  - list
+  - watch
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: k8s-ra-event-watcher
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: event-watcher
+subjects:
+- kind: ServiceAccount
+  name: events-sa
+  namespace: default
+```
```

Comment:
Add a line between lines 67 and 68. Our markdown processor strongly prefers there being a blank line between the end of a codeblock and new text. (Otherwise it does bad formatting things.)